WebContent/login.html

```diff
@@ -69,4 +69,9 @@ <h3 class="text-center sub-heading translate" data-i18n="Welcome Back">Welcome B
     
     <div id="footer-outer"></div> 
   </body>
+  <script src="resources/lib/url/url.min.js"></script>  
+<script src="resources/i18next.min.js" ></script>
+<script src="resources/jquery-i18next.min.js" ></script>
+<script src="resources/i18nextXHRBackend.js"></script>
+<script src="resources/global.js"></script>
 </html>
```

Comment:
Please move the script references to the top of the HTML page with the rest of the script references (code readibility / maintainability issue)